prime-router/metadata/valuesets/monkeypox.valuesets

```diff
@@ -0,0 +1,16 @@
+# Some valuesets
+---
+- name: monkeypox/specimen_type
+  system: SNOMED_CT
+  version: 2.67
+  referenceUrl: ""
+  values:
+    - code: "257261003"
+      display: "Swab (specimen)"
+
+- name: monkeypox/test_result
```

Comment:
OK for now.   

Still need these test_result codes:



Inconclusive | 419984006
Equivocal | 42425007
Not detected | 260415000
Test not done | 373121007

Also need these LOINC test_performed codes:



Test | Test Performed   LOINC Long Name | LOINC Code
-- | -- | --
Orthopoxvirus | Orthopoxvirus DNA [Presence] in Specimen by NAA with probe detection | 41853-3
Non-variola Orthopoxvirus | Orthopoxvirus non-variola DNA [Presence] in Specimen by NAA with probe detection | 100434-0
Monkeypox Virus | Monkeypox virus DNA [Presence] in Specimen by NAA with probe detection | 100383-9